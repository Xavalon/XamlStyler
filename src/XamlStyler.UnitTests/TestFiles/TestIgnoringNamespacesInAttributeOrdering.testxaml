<Window x:Class="WpfTests.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:local="clr-namespace:WpfTests"
        mc:Ignorable="d">
    <Grid>
        <TextBlock
		    d:Foreground="White"
			Text="Hello"
			DataContext="{x:Null}"
			d:Background="White"
			Foreground="Black"
			Background="Black"
			d:Text="Design time"
			d:DataContext="{x:Null}" />
    </Grid>
</Window>
