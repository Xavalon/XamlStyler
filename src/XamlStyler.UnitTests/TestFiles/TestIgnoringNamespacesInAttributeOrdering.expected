<Window
    x:Class="WpfTests.MainWindow"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:local="clr-namespace:WpfTests"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    mc:Ignorable="d">
    <Grid>
        <TextBlock
            Background="Black"
            d:Background="White"
            DataContext="{x:Null}"
            d:DataContext="{x:Null}"
            Foreground="Black"
            d:Foreground="White"
            Text="Hello"
            d:Text="Design time" />
    </Grid>
</Window>
